packages/slice-machine/src/legacy/lib/models/common/widgets/Link/Form.jsx

```diff
@@ -11,7 +11,7 @@ import { DisplayTextCheckbox } from "./components";
 // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
 const FormFields = {
   ...DefaultFields,
-  allowTargetBlank: CheckBox("Allow target blank", false, false),
+  allowTargetBlank: CheckBox("Allow target blank", false, true),
```

Comment:
💡 #idea

Even if not really used, I would change the `defaultValue` param in Checkbox function `packages/slice-machine/src/legacy/lib/forms/fields.ts`. 

So it's clear everywhere what default we want.

Reply:
Or even directly used the fallback

Reply:
why? that'd assume that we want all checkboxes checked by default, no?

Reply:
Hum, it's because I saw now we always pass true, but not really required of course.

Reply:
You're right that it's 2 different things